### PR TITLE
Documentation for Taxon table

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #1642

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 0 sec

**Comment by ketankartoza** [comment]
@Jeremy-Prior Please review. Thanks!

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 15 sec

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 27 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 54 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 20 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 49 sec

File: docs/src/administrator/manual/django-table.md [hunk 1]
```diff
@@ -69,3 +69,23 @@ This table is used for storing, generating, and deleting the TOTP device.
...
+
+## Taxon Table
+
+![Taxon Table](./img/django-table-8.png)
+
+The Taxon table within the Django Admin interface provides administrators with the tools to manage taxonomic information efficiently.
+
+1. **Add Taxon**: Clicking on the `ADD TAXON` button enables administrators to add a new taxon. To learn more about adding a new taxon, refer to the detailed documentation on [adding a taxon](django-add-data.md).
+
+2. **Search Functionality**: Administrators can utilize the search functionality to find specific taxonomic records.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-table.md:81]:
```suggestion
2. **Search Functionality**: Administrators can utilise the search functionality to find specific taxonomic records.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 52 sec

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 22 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 51 sec

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 38 sec

**Pull request merged**